apps/friesian-server-helm/README.md

```diff
@@ -0,0 +1,42 @@
+# Friesian server deploying with Helm Chart
+
+This example demonstrates how to deploy a Friesian recommender server with Helm Chart on a
+Kubernetes cluster.
+
+## Prepare environments
+
+* AVX512 instruction support on nodes that deploy the recall server
+* [Preparation steps](./preparation) or a PV contains resource files
+* [Helm](https://helm.sh) 3.2.0+
+* Kubernetes 1.19+
+
+## Deploy Friesian server
+
+You can get the full installation document from [here](./friesian-helm/README.md).
+
+## TL;DR
```

Comment:
May be use a formal tile instead of TL;DR?

Reply:
agree

Reply:
modified to 'Quick Installation'